### DISCUSSION TITLE
Focus to input field not working in Cefsharp

### DISCUSSION THREAD

**Discussion by phuocminh6464** [cefsharp/CefSharp#4840] [General]
- **What version of the product are you using?**
125.0.210

- **What architecture x86 or x64?**
x64

- **On what operating system?**
Windows 10 and Windows 11 (touch screen)

- **Are you using WinForms, WPF or OffScreen?**
WPF application

- **What steps will reproduce the problem?**
1. A web url has only 1 input field and it's focused
2. Open this url by web browser, users able to input data by physical keyboard
3. Run this url in WPF app by Cefsharp, not able to input data (whereas the cursor is focus) unless users touch/click mouse to input field

Anyone can help me? Thanks.

**Comment by amaitland** [comment]
How did you get the ` ChromiumWebBrowser` focus? Did you give it explicit focus? e.g. using the `FocusManager`?

https://github.com/cefsharp/CefSharp.MinimalExample/blob/master/CefSharp.MinimalExample.Wpf/MainWindow.xaml#L9

**Reply by amaitland** [reply]
Please test with https://github.com/cefsharp/CefSharp.MinimalExample/ and confirm the behaviour reproduces there.

**Reply by phuocminh6464** [reply]
> How did you get the ` ChromiumWebBrowser` focus?

I don't set focus from WPF app, the web page will focus to input field.
By the way, the `FocusManager` is working for me. Thanks you so much.
